store/sqlstore/post_store.go

```diff
@@ -1108,19 +1107,9 @@ func (s *SqlPostStore) GetPostsCreatedAt(channelId string, time int64) store.Sto
 
 func (s *SqlPostStore) GetPostsByIds(postIds []string) store.StoreChannel {
 	return store.Do(func(result *store.StoreResult) {
-		keys := bytes.Buffer{}
-		params := make(map[string]interface{})
-		for i, postId := range postIds {
-			if keys.Len() > 0 {
-				keys.WriteString(",")
-			}
-
-			key := "Post" + strconv.Itoa(i)
-			keys.WriteString(":" + key)
-			params[key] = postId
-		}
+		keys, params := MapStringsToQueryParams(postIds, "Post")
 
-		query := `SELECT * FROM Posts WHERE Id in (` + keys.String() + `) and DeleteAt = 0 ORDER BY CreateAt DESC`
+		query := `SELECT * FROM Posts WHERE Id in ` + keys + ` and DeleteAt = 0 ORDER BY CreateAt DESC`
```

Comment:
I can change the `in` to `IN`, and I made it so that the new `MapStringsToQueryParams` function returns the keys surrounded by brackets since I couldn't see a situation where we wouldn't use a function like that without brackets